site/docs/webrtc/overview.md

```diff
@@ -66,44 +51,42 @@ More can be found on this in our [quickstart guide](/docs/webrtc/quickstart).
 
 ## Sessions
 
-Sessions are a concept that allows you to easily group multiple Participants together - depending on your use case, this may be analogous to something like a meeting, appointment, call, or event. Participants and their streams are associated with one another via subscriptions, described below.
+Sessions are a concept that allow you to group multiple participants together - depending on your use case, this may be analogous to something like a meeting, appointment, call, or event. participants and their streams are connected to one another via subscriptions, described below.
 
 ### Limitations
 
-There is a limit of 20 participants in a session. This includes all participants, whether they are Audio-only, Audio & Video, or dialed-in participants.
-
-### Session Tags
+There is a limit of 20 participants in a session.
 
-You can add a tag to a session to help you identify them with friendly names. Session Tag information is passed on to your Billing Reports, which can facilitate billing of clients if you do pass-through billing or auditing. These can also be used to allow for easier understanding or grouping of sessions.
+### Participant-Session Membership
 
-Example uses of a Participant Tags:
+A participant by itself cannot receive media. It must first be associated with, or a member of, a session. Only then can it receive media from other session participants. The (Add Participant to Session Endpoint)[https://new.dev.bandwidth.com/apis/webrtc#operation/addParticipantToSession] is used to add a participant to a session. The target participant and session are indicated by ID in the URL path:
```

Comment:
To make it super clear, maybe change the last sentence to something like:

"The participant being added and the target session are each indicated by ID in the URL path:"